### PR TITLE
Rex fix formatting nits

### PR DESCRIPTION
**Pull request by RexJaeschke**
@jskeet This one can now be merged.

### CODE REVIEW

**Comment by jskeet** [comment]
Label removed as this is a PR that Rex uses to fix typos periodically.

File: standard/classes.md [hunk 1]
```diff
@@ -2659,7 +2659,7 @@ For `abstract` and `extern` properties, the *accessor_body* for each accessor sp
...
 
-A `set` accessor corresponds to a method with a single value parameter of the property type and a `void` return type. The implicit parameter of a `set` accessor is always named value. When a property is referenced as the target of an assignment ([§12.18](expressions.md#1218-assignment-operators)), or as the operand of `++` or `–-` ([§12.7.10](expressions.md#12710-postfix-increment-and-decrement-operators), [§12.8.6](expressions.md#1286-prefix-increment-and-decrement-operators)), the `set` accessor is invoked with an argument that provides the new value ([§12.18.2](expressions.md#12182-simple-assignment)). The body of a `set` accessor shall conform to the rules for `void` methods described in [§15.6.11](classes.md#15611-method-body). In particular, return statements in the `set` accessor body are not permitted to specify an expression. Since a `set` accessor implicitly has a parameter named `value`, it is a compile-time error for a local variable or constant declaration in a `set` accessor to have that name.
```
Comment (jskeet) [inline standard/classes.md]: I can't spot the difference here... any ideas?

File: standard/classes.md [hunk 2]
```diff
@@ -4258,9 +4258,9 @@ It is a compile-time error for the formal parameter list of an async function to
...
+An async function has the ability to suspend evaluation by means of await expressions in its body. Evaluation may later be resumed at the point of the suspending await expression by means of a ***resumption delegate***. The resumption delegate is of type `System.Action`, and when it is invoked, evaluation of the async function invocation will resume from the await expression where it left off. The ***current caller*** of an async function invocation is the original caller if the function invocation has never been suspended or the most recent caller of the resumption delegate otherwise.
```
Comment (jskeet) [inline standard/classes.md]: Why have we removed the link here?

**Review by jskeet — COMMENTED** [review]
Mostly fine, although there are conflicts.

File: standard/classes.md (hunk 1 above)
Comment (jskeet) [reply standard/classes.md]: Ah, it's `value` in the second sentence.

**Review by jskeet — COMMENTED** [review]
There's still one comment about link removal, but once that's sorted I'd like to get this merged, to avoid it getting stale. (It's quite old now.) I don't know whether it's already out of date with respect to section references.

File: standard/classes.md (hunk 2 above)
Comment (BillWagner) [reply standard/classes.md]: When resolving conflicts, I put the link back in. @RexJaeschke Does that make sense to you?

**Review by BillWagner — APPROVED** [review]
This LGTM.

@RexJaeschke See the one comment from Jon and I regarding removing a link.

Once we're clear there, I'll merge this.

**Comment by RexJaeschke** [comment]
@BillWagner The link you restored looks good, Bill. But the link-verifier failure reported above appears to be in README.md, which is not something I'm touching.

**Comment by BillWagner** [comment]
> But the link-verifier failure reported above appears to be in README.md, which is not something I'm touching.

We need to fix that in our tooling. The links in the readme are generated by our tool. So, we don't care if they aren't correct when we remove or replace a clause. (The tool will fix it).

I'll work with @Youssef1313 to submit a PR so we can ignore bad links in that file.

**Comment by Youssef1313** [comment]
@BillWagner It looks like you already fixed the failure here in #514. If the workflow was given a re-run, it would have succeeded.

**Comment by Youssef1313** [comment]
Just need to make sure the tool that generates README.md doesn't overwrite the change in #514

**Comment by BillWagner** [comment]
@Youssef1313 Yes, this condition is self-correcting.

See #484 it reappears every time we add or remove a section, until that PR gets merged, and then the action runs to regenerate the TOC. It's a lower priority, but in time, I'd still like all the PRs to have green CI results.

**Comment by Youssef1313** [comment]
While #514 was a good step in getting CI green. It's possible this fail in the future for characters that are not handled correctly (currently the ellipse was special-cased).

A more stable fix would be to replace this logic:

https://github.com/dotnet/csharpstandard/blob/133725ebb791804fba0e8ebc10b5618682d70269/tools/StandardAnchorTags/TocSectionNumberBuilder.cs#L154-L159

with

https://github.com/xoofx/markdig/blob/0cfe6d7da48ea6621072eb50ade32141ea92bc35/src/Markdig/Helpers/LinkHelper.cs#L100-L113

I opened https://github.com/dotnet/csharpstandard/pull/520

**Pull request merged**